src/transformer/section-category.ts

```diff
@@ -220,6 +220,33 @@ export const chooseSecType = (sectionCategory: SectionCategory): SecType => {
   }
 }
 
+export const chooseCoreSectionBySection = (section: string): string => {
+  switch (section) {
```

Comment:
since the transformer doesn't have access to the Section-Category.json from the API, this made sense, @mnatsheh what do you think?